src/Data/Function.purs

```diff
@@ -77,6 +81,20 @@ applyFlipped x f = f x
 -- | ```
 infixl 1 applyFlipped as #
 
+-- | Applies the function `f` to its argument `n` times.
+-- |
+-- | If n is less than or equal to 0, the function is not applied.
+-- |
+-- | ```purescript
+-- | applyN (_ + 1) 10 0 == 10
+-- | ```
+applyN :: forall a. (a -> a) -> Int -> a -> a
+applyN f = go
+  where
+  go n' acc
```

Comment:
Could just use `n` here instead of `n'` now that it won't shadow?

Reply:
Will do.